.github/ISSUE_TEMPLATE/design-user-story-issue.md

```diff
@@ -0,0 +1,19 @@
+---
+name: Design User Story
+about: This template provides a basic structure for user stories that detail designs.
+title: '[Design] <title>'
+labels: Design, Needs Triage
+---
+
+# Description
+
+
+# Acceptance criteria
+
+- [ ] This is something that can be verified to show that this user story is satisfied.
+- [ ] Architectual Decisions documented in a decision log
+
+# Sprint Ready Checklist 
+- [ ] Acceptance criteria defined 
+- [ ] Team understands acceptance criteria 
```

Comment:
Consider the word `reviewed` over `understands`.  Relatedly, how many people need to review/understand to check this box? 

```suggestion
- [ ] Acceptance criteria reviewed by engineering team
```